system/Router/RouteCollection.php

```diff
@@ -1490,4 +1491,21 @@ public function shouldUseSupportedLocalesOnly(): bool
     {
         return $this->useSupportedLocalesOnly;
     }
+
+    /**
+     * Loads main routes file and discover routes.
+     *
+     * Loads only once unless reset.
+     */
+    public function loadRoutes(string $routesFile = APPPATH . 'Config/Routes.php'): void
```

Comment:
All of the calls you replaced look like this:
```php
    $routes = Services::routes();
    $routes->loadRoutes();
```
Which makes me think this would be better off returning `$this` so those can become: `$routes = Services::routes()->loadRoutes()`.

Reply:
Since `loadRoutes()` is essentially an initialization process, it is better to group it with `Services::routes()->loadRoutes()`.

Actually, I would like to call `loadRoutes()` in the constructor, but we can't because `Routes.php` has `$routes = Services::routes();`.